src/components/organisms/SchedulePageModal/SchedulePageModal.utils.ts

```diff
@@ -0,0 +1,97 @@
+import {
```

Comment:
Potentially just call this file `utils.ts` and leave it in this folder? Not sure it needs such a verbose name.

Reply:
renamed: https://github.com/sparkletown/sparkle/pull/1302/commits/6c79ed94ad0acdbf8061bba26c4e73f7912ed2de